patch/0001-hwmon-supporting-emc2302.patch

```diff
@@ -0,0 +1,38 @@
+From 291cfd2e11a31164aaf9eeff38447fc553c74dbd Mon Sep 17 00:00:00 2001
+From: Natarajan Subbiramani <natarajan.subbiramani.ext@nokia.com>
+Date: Mon, 20 Mar 2023 18:22:33 +0000
+Subject: [PATCH] Adding EMC2302 support
+
+Existing code supports only 2305 and this patch enables the driver to 
+support 2302 model as well.
+
```

Comment:
Please amend the commit message, and add a reference to a document/specification, why the same driver can be used.